README.md

```diff
@@ -1,7 +1,300 @@
+
+### FDR suggestions & common mistakes
+
+The extreme flexibility of xiFDR requires some careful use. Here are some of our suggestions and ways to spot that the FDR calculation may not be giving sensible results. When in doubt, keep it simple! 
+
+#### Tips & tricks
+- For searches with cleavable crosslinkers such as DSSO, set prefilters (see below) on the number of peptide doublets and boost on that parameter
+- For searches with unspecific crosslinkers like SDA, include prefilters (or boost) on minimum number of fragments. Include also a prefilter on "fragment unique crosslinked matched conservative" and exclude consecutives. This ensures that only spectral matches with at least one fragment proving a crosslinked peptide pair are considered.
+
+#### Watch out for:
+- The program warns if there aren't sufficient targets to estimate FDR accurately. If this warning refers to the level of analysis you are interested in, there is likely almost nothing in the data. Unless prefilters were set way too stringent.
+- The FDR calculation rests on the assumption that target-decoy pairs have twice the chance of random matching than decoy-decoy pairs (TD+DT vs DD). If in your results you have more decoy-decoy (DD) than target-decoy (TD), your FDR evaluates to a negative number and becomes meaningless. This may be a sign that there are no crosslinks in the datasets, or that prefilters are not working as intended.
+
+
+### Results summary
+After the calculation is complete (check the log tab, the lower bottom left of the window or the "calculate" button becoming clickable again), the results tab presents a summary of all matches passing the FDR threshold with the number of decoys and target-decoys in each category. Remember the ratio of targets and decoys passing determines the accuracy of the FDR estimation.
+
+### Writing out search results
+
+Results may be written out in xiFDR .csv format (csv tab) or in mzIdentML1.2.0 format. We advise to do both every time, as the summary file is useful to keep track of what was done, and the mzIdentML for later deposition or upload to [xiView.org](https://www.xiview.org).
+
+#### csv output 
+will generate several files:
+
+- Summary file: a file containing a summary of the FDR calculation, results and all settings
+- CSM file: a file containing all crosslink spectra matches passing the threshold. This file can be uploaded to xiview.org for visualization
+- peptide pairs file: a file containing all peptide pairs passing the threshold
+- protein groups file: all protein groups passing the threshold (i.e. including ambiguity when a peptide cannot be assigned to an individual protein)
+- Links file: all the crosslinked residue pairs passing the threshold
+- PPI file: all the protein-protein interactions passing the thresold.
+
+Each file contains information about the FDR, local FDR (if enabled), posterior error probability, target/decoy nature of each match, as well as the peakfile of origin.
```

Comment:
```suggestion
Each file contains information about the FDR, posterior error probability (local FDR), target/decoy nature of each match, as well as the peakfile of origin.
```

Reply:
we are using local FDR everywhere, why would we bring in PEP? is it in the headers?

Reply:
in the output file the column is called PEP - you are right. It should be unified

Reply:
then for now i commit your change (anyways in the terminology i write that they are the same)